lib/struct.js

```diff
@@ -184,6 +186,16 @@ function toString () {
 }
 
 /**
+ * Change struct padding mode.
+ *
+ * @api public
+ */
+function usePack (flag) {
```

Comment:
@TooTallNate I will try to second way, but this library direct export `Struct`
So, I will use `Struct.Packed` instead `PackedStruct`, and remove `usePack`, `StructType.isPacked` assign using arguments,  is this ok?
If not, i will just remove `usePack` method